src/main/java/name/valery1707/kaitai/KaitaiGenerator.java

```diff
@@ -308,9 +338,12 @@ public Path generate(Logger log) throws KaitaiException {
 		}
 
 		ProcBuilder builder = process(log)
-			.withArgs("--target", "java")
-			.withArgs("--outdir", output.toFile().getAbsolutePath())
-			.withArgs("--java-package", getPackageName());
+				.withArgs("--target", "java")
+				.withArgs("--outdir", output.toFile().getAbsolutePath())
+				.withArgs("--java-package", getPackageName());
```

Comment:
Please revert the indentation changes
```suggestion
			.withArgs("--target", "java")
			.withArgs("--outdir", output.toFile().getAbsolutePath())
			.withArgs("--java-package", getPackageName());
```